conga-aem-definitions/src/main/templates/aem-dispatcher-cloud/conf.d/available_vhosts/tenant.vhost.partials.hbs

```diff
@@ -98,11 +98,16 @@ ServerSignature Off
 
 {{~#block "cacheControl"}}
 # Cache-control
-Header set Cache-Control "public, must-revalidate"
```

Comment:
removing this default setting and adding it to the list of defaults breaks the backward compatibility. although it like the improvement from this PR, this might create subtle problems for people updating just to the latest version, if they define their own list of cache control settings without merging.

Reply:
@stefanseifert you are right. I have readded the default but made it configurable.